docs/update-cli.md

```diff
@@ -0,0 +1,26 @@
+# Update OpenAPI CLI
+
+If you've already installed OpenAPI CLI, you'll want to keep it as up to date as possible. Staying on top of versions ensures that you get the latest functionality and bug fixes. Depending on how you originally installed OpenAPI CLI, there are two ways to update it.
+
+:::success Tip
+Before you begin, ensure that you have the latest version of npm installed.
+:::
+
+## How to update your openapi-starter installation
+
+If you installed OpenAPI CLI using the [openapi-starter template](https://github.com/Redocly/openapi-starter) follow these steps:
```

Comment:
This should be about project-level installations. Not about the openapi-starter template. Anyone can add a project-level installation in other ways too. 

Example: https://github.com/Rebilly/api-definitions/blob/main/package.json#L5